### ISSUE TITLE
Python Modules use default value over input file value

### ISSUE THREAD

**Issue by FlanFlanagan** [cyclus/cyclus#1441]
If module input field has a default value the default value is always taken even if a value is provided in the input file.

**Comment by scopatz** [comment]
Can you please give a more detailed example.

**Comment by FlanFlanagan** [comment]
Sure. 

If I set this as a variable on the module. 

```
    demand_ts = ts.Int(
        doc = "The number of timesteps between demand calls by the agent", 
        tooltip = "The number of timesteps between demand calls by the agent", 
        uilabel = "Demand Timestep",
        default = 1
    ) 
```
And then put this inside the input file

```
  <facility>
    <config>
      <DemandFac>
        <supply_commod>power_rx</supply_commod>
        <demand_commod>fuel_rx</demand_commod>
        <demand_rate_max>2000.</demand_rate_max>
        <demand_rate_min>2000.</demand_rate_min>
        <supply_rate_max>1000.</supply_rate_max>
        <supply_rate_min>1000.</supply_rate_min>
        <demand_ts>18</demand_ts>
        <proto>Reactor</proto>
      </DemandFac>
    </config>
    <name>Reactor</name>
  </facility>
```

If I have the module print out demand_ts using, 
`print(self.demand_ts)`

The return value is 1.

**Comment by scopatz** [comment]
Thanks for reporting @FlanFlanagan! Potential fix in #1442
